includes/class-wc-stripe-webhook-handler.php

```
@@ -184,6 +201,65 @@ public function check_for_webhook() {
+	}
+
+	/**
+	 * The connected Stripe account ID for the active mode.
+	 *
+	 * @return string Account ID (e.g. `acct_123`), or an empty string when unknown.
+	 */
+	protected function get_connected_account_id(): string {
+		$account_data = WC_Stripe::get_instance()->account->get_cached_account_data();
+
+		return isset( $account_data['id'] ) ? (string) $account_data['id'] : '';
+	}
```

PHPMD (2.15.0)

[error] The method get_connected_account_id is not named in camelCase. (undefined)

(CamelCaseMethodName)